src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java

```diff
@@ -184,4 +185,43 @@ public void retrieve_an_organisations() {
                 times(1)).findAll();
 
     }
+
```

Comment:
I don't quite understand what you are testing here? Your test should check that an organisation has been updated. You are checking that the setter has been called. But should use be more specific to check that a status of PENDING is now ACTIVE etc.

Reply:
This is not done by me . This is retrieve_an_organisations() done by Prashanth.